staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go

```diff
@@ -45,7 +45,7 @@ var ErrAPIServiceNotFound = errors.New("resource not found")
 type SpecAggregator interface {
 	AddUpdateAPIService(apiService *v1.APIService, handler http.Handler) error
 	UpdateAPIServiceSpec(apiServiceName string) error
-	RemoveAPIService(apiServiceName string) error
```

Comment:
you introduced a special error. This is becoming part of the interface and needs documentation. Or better: get rid of the error and use `found bool, err error`

Reply:
updated

Reply:
just a note that adding a return parameter will probably make backporting slightly more involved